### PR TITLE
Represent foreign keys as 'field'. Fixes #361

### PR DESCRIPTION
**Pull request by steven-mercatante**

### CODE REVIEW

**Comment by nmohoric** [comment]
:+1:

File: rest_framework_swagger/docgenerator.py [hunk 1]
```diff
@@ -365,7 +369,7 @@ def _get_serializer_fields(self, serializer):
                     f['type'] = 'array'
                     if field_serializer:
                         f['items'] = {'$ref': field_serializer}
-                    elif data_type in BaseMethodIntrospector.PRIMITIVES:
```
Comment (cvitan) [inline rest_framework_swagger/docgenerator.py:368]: What effect does the removal of this conditional have?

Comment (steven-mercatante) [inline rest_framework_swagger/docgenerator.py:368]: Removing this conditional lets Swagger UI output `array(field)`. If the conditional was left in place, we would just see `array`. The conditional was preventing anything that's not in `BaseMethodIntrospector.PRIMITIVES` from being output. I could've added a `field` key to `PRIMITIVES`, but I didn't think it qualifies as a primitive.

For reference, `PRIMITIVES` looks like this:

```
    PRIMITIVES = {
        'integer': ['int32', 'int64'],
        'number': ['float', 'double'],
        'string': ['string', 'byte', 'date', 'date-time'],
        'boolean': ['boolean'],
    }
```

Comment (cvitan) [inline rest_framework_swagger/docgenerator.py:368]: Great, thanks for elaborating on that. You might want to add that to the PR description.

**Comment by cvitan** [comment]
:+1:

**Pull request merged**